apps/gro/fields/address.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  'address': {
+    className: ['address'],
+    options: [''],
+    legend: {
+      className: 'visuallyhidden',
+      value: 'fields.address.label'
+    },
```

Comment:
what's the reason we are using a legend and a label with the same value? And how come we are giving the legend the `visuallyhidden` class?